.github/workflows/python-app.yml

```diff
@@ -0,0 +1,36 @@
+name: dbt-junitxml
```

Comment:
@si-aliaksandra by the way, would be really great to update the README section about this fork section, since now we're much more ahead than initial repo